pandas/tools/tests/test_tile.py

```diff
@@ -288,7 +288,10 @@ def test_qcut_duplicates_bin(self):
 
     def test_single_bin(self):
         # issue 14652
-        expected = Series([0, 0])
+        # Explicit dtype since Series produces int64 for ints, while cut
+        # (due to numpy.searchsorted) would use int32 on i386, so let's assure
+        # correct default to the architecture int
```

Comment:
https://github.com/pandas-dev/pandas/blob/master/pandas/tools/tile.py#L218

needs an ``.astype('int64', copy=False)`` as these can get returned to the user in some paths; and they are 32-bit on 32-bit :< (while 64bit on others)